thunder/tests/opinfos.py

```diff
@@ -1678,6 +1678,23 @@ def gen(op, device, dtype, requires_grad):
 elementwise_unary_ops.append(leaky_relu_opinfo)
 
 
+logsigmoid_opinfo = OpInfo(
+    ltorch.logsigmoid,
+    dtypes=(datatypes.floating,),
+    sample_input_generator=elementwise_unary_generator,
+    torch_reference=torch.nn.functional.logsigmoid,
+    test_directives=(
+        # test tols are too tight for these half precision tests
```

Comment:
The test tolerances can be relaxed by using a custom comparator, like here:

https://github.com/Lightning-AI/lightning-thunder/blob/9c84e235d6b6deabb62a9bfd21b9ecc21f6d18f2/thunder/tests/opinfos.py#L1815

Reply:
Yes, I can do that, but it seems liked the tolerances must be relatively high.  I copied this directive from the sigmoid opinfo figuring that it was decided that it was preferable to skip the test rather than use such large tolerances.

Interestingly, the naive implementation of `log(sigmoid(a)` passes the tests with tolerances of (1e-2, 1e-2) while the SciPy-like implementation of `where(a < 0, -log1p(exp(-a)), a - log1p(exp(a)))` requires (1e-1, 1e-1).  This is unexpected as the justification for the SciPy implementation is greater accuracy, but maybe that doesn't necessarily apply uniformly across datatypes.

Reply:
That makes sense! I believe SciPy's implementation (and PyTorch's implementation) are designed to improve numerical accuracy, and we are testing for consistency with PyTorch's implementation